test/lib/Chainweb/Test/Pact5/Utils.hs

```diff
@@ -251,3 +260,17 @@ getTestLogger = do
     logLevel <- getTestLogLevel
     return $ genericLogger logLevel (testLogFn logLevel)
 
+-- usually we don't want to check the module hash
```

Comment:
Why do we not want to? I kind of do appreciate more checks and balances on module hashes changing suddenly in pact 5. We do have checks in the pact 5 repo, but more are always appreciated.

Reply:
Done.